src/main/java/com/nyxei/pollingus/controllers/Digester.java

```diff
@@ -0,0 +1,25 @@
+package com.nyxei.pollingus.controllers;
+
+import com.nyxei.pollingus.service.Survey;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class Digester {
+    public Survey survey;
+
+    public static void  lectureMd(String fileReader){
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fileReader));
+            String line = reader.readLine();

```

Comment:
We should aim at consistency with the used language. English is the preferred one, but if you choose french, use it everywhere.

Reply:
nouveau nom de la méthode readMd